oleesir/diaryReload
# #160951989 Make the pages responsive and interactive

#### What does this PR do?
it makes the app responsive and the profile and reminder interactive 

#### Description of Task to be completed?
* add responsiveness to the app

#### How should this be manually tested?
* Navigate to the UI folder and open all html files in your browser
* You will see all the template

#### What are the relevant pivotal tracker stories?
[#160951989](https://www.pivotaltracker.com/story/show/160951989)

#### Any background context you want to add?
N/A
#### Screenshots
<img width="1280" alt="screen shot 2018-10-03 at 9 30 43 pm" src="https://user-images.githubusercontent.com/17679511/46437621-b2285200-c753-11e8-8619-5ee553170b0e.png">
